inst/rstudio/templates/project/resources/bs4_book/_output.yml

```
@@ -0,0 +1,5 @@
+bookdown::bs4_book:
+  css: bs4_style.css
+  theme:
+    primary: "#096B72"
+  repo: https://github.com/johndoe/bookdown-demo
```

`johndoe` is an existing username. We should probably use one that doesn't exist, e.g.,

```suggestion
  repo: https://github.com/USERNAME/bookdown-demo
```

---

Oh that is true ! Good thinking ! 

Should we use our own repo otherwise https://github.com/rstudio/bookdown-demo ? 

@apreshill I let you decide.